Adding apps to pbcs (#1203)
spryker/spryker-docs: docs/pbc/all/usercentrics/integrate-usercentrics.md

That's it. You have integrated the Usercentrics app into your store. The app changes to *Connected* on the apps catalog page. Now, the cookie consent dialogs should be displayed on every page of the Storefront.

## Global settings configuration

> There are three ways to integrate Usecentrics: by direct integration, by the Usercentrics Smart Data Protector, and integration with Google Tag Manager. In the Spryker Back Office, you can select either Smart Data Protector or Google Tag Manager. At the same time, the Smart Data Protector is the preferred and default setting.

Reviewer: ```suggestion
There are three ways to integrate Usecentrics: a direct integration,  using the Usercentrics Smart Data Protector, or using the Google Tag Manager. In the Spryker Back Office, you can select either Smart Data Protector or Google Tag Manager. At the same time, the Smart Data Protector is the preferred and default setting.
```

Author: "a direct integration,  using the Usercentrics Smart Data Protector" might be confusing, as it can be understood as one option. However, direct integration and Usercentrics Smart Data Protector are two different options, we should emphasize this.

{% endinfo_block %}

### Direct integration

> The direct integration requires quite a bit of manual work by developers.

Reviewer: Not sure why we need to mention this. Would a `manual integration` be a better match for this way?

Author: Usercentrics calls this "direct implementation", and our engineers also prefer this term :)

For more details about the Usercetrics direct integration, see the [Usercentrics documentation](https://docs.usercentrics.com/#/direct-implementation-guide).

### Smart Data Protector

> The [Smart Data Protector](https://docs.usercentrics.com/#/smart-data-protector) blocks the data processing services you added to your store and activates them only for customers that gave their consent to do so. The [Smart Data Protector](https://docs.usercentrics.com/#/smart-data-protector) is the preferred and default setting to configure Usercentrics using Spryker. You can apply it if you don't use a third-party tracking management tool yet such as [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager) or others and prefer a code-free and therefore the most convenient approach to configure Usercentrics.

Reviewer: Please move the last two sentences to the start of the document where you explain that there are three ways to integrate Usercentrics. The paragraph of each integration should explain its individual advantages, but the comparison should be at the start of the doc, so the reader can decide which way to go with.

In this paragraph, you can say something like `If you are not using a third-party tracking management tool like [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager) and want a code-free integration, we recommend integrating Usercentrics via Smart Data Protector.

{% endinfo_block %}

### Google Tag Manager

If you already have the [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager) integrated into your store, to use the technology legally, you need the cookie consent dialog to appear in your stores. You can achieve that by using the Google Tag Manager with Usercentrics.

> To use Usercentrics with the Google Tag Manager, make sure that:

Reviewer: ```suggestion
To use Usercentrics with the Google Tag Manager, make sure the following applies:
```

> - The Usersentrics data processing services match the tracking tools you have in the [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager). For this, in the [Usercentrics Admin Interface](https://admin.usercentrics.eu/), go to **Service Settings&nbsp;<span aria-label="and then">></span> Data Processing Services** and define the same data processing services that you have in the Google Tag Manager.

Reviewer: ```suggestion
- The Usersentrics data processing services match the tracking tools in the [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager). To do that, in the [Usercentrics Admin Interface](https://admin.usercentrics.eu/), go to **Service Settings&nbsp;<span aria-label="and then">></span> Data Processing Services** and define the same data processing services that you have in the Google Tag Manager.
```
